Recruit-Training/Embedded-Recruit-Training/Core/Src/GreenLedToggleTask.c

```diff
@@ -7,11 +7,39 @@ void greenLedToggleTask(void const* arg)
 
     osMutexId_t* canMutex = (osMutexId_t*)arg;
 
+    static const uint32_t GREEN_LED_STATUS_STDID = 0xDDD;
+    static const uint32_t GREEN_LED_TOGGLE_FREQ = 100;
+
     for (;;)
     {
         //TODO: Add GREEN_LED_TOGGLE_FREQ to prevWakeTime
+        prevWakeTime += GREEN_LED_TOGGLE_FREQ;
         osDelayUntil(prevWakeTime);
-        //TODO: Check blue toggel flag and toggle blue LED
+
+        //TODO: Check green toggle flag and toggle green LED
+        if(greenLedToggleFlag) 
+        {
+            HAL_GPIO_TogglePin(GPIOA, LED_GREEN_Pin);
+        } 
+
         //TODO: Send CAN message indicating current state of LED
+        uint8_t GREEN_LED_STATUS_ID = HAL_GPIO_ReadPin(GPIOA, LED_GREEN_Pin);
```

Comment:
All capital case is usually reserved for constants that won't change. This should just be a normal camelCase variable.